js/main.js

```diff
@@ -21,8 +21,100 @@ $entryForm.addEventListener('submit', function () {
   entryData.photoUrl = $entryForm.elements.photoUrl.value;
   entryData.notes = $entryForm.elements.notes.value;
   entryData.entryId = data.nextEntryId;
-  data.entries.push(entryData);
+  data.entries.unshift(entryData);
   data.nextEntryId++;
+  $newEntries.prepend(renderData(entryData));
   $picView.setAttribute('src', 'images/placeholder-image-square.jpg');
   $entryForm.reset();
+
 });
+
+function entryDisplay(event) {
+  for (var i = 0; i < data.entries.length; i++) {
+    var $newEntry = renderData(data.entries[i]);
+    $newEntries.appendChild($newEntry);
+  }
+  for (var j = 0; j < $view.length; j++) {
+    if ($view[j].getAttribute('data-view') === data.view) {
+      $view[j].className = ' view ';
+    } else {
+      $view[j].className = 'view hidden';
+    }
+  }
```

Comment:
Once you revise your `viewChange` function in the way I mentioned, you should be able to just use the `viewChange` function here.